corehq/apps/accounting/async_handlers.py

```diff
@@ -43,10 +43,10 @@ class FeatureRateAsyncHandler(BaseRateAsyncHandler):
 
     @property
     def create_response(self):
-        if Feature.objects.filter(name=self.name).count() > 0:
+        if Feature.objects.filter(name=self.name):
```

Comment:
fyi I think `.exists()` can be slightly more efficient. see [bool section below here](https://docs.djangoproject.com/en/1.10/ref/models/querysets/#when-querysets-are-evaluated)